README.md

```diff
@@ -4,202 +4,195 @@ The Code of Conduct establishes and communicates the commitment of the ml5.js co
+
+As part of the ml5.js Code of Conduct, we pledge to design, build, and use ml5.js in a way that centers the aforementioned marginalized bodies and communities first, striving to always confront our biases, privileges, and ignorances for our own good and society at large.
 
 **In practice:**
 
-- We are not code snobs. We do not assume knowledge or imply there are things that somebody should know.
-- We insist on actively engaging with requests for feedback regardless of their complexity.
-- We welcome newcomers and prioritize the education of others. We strive to approach all tasks with the enthusiasm of a newcomer. Because we believe that newcomers are just as valuable in this effort as experts.
+- We welcome newcomers and prioritize the education of others. We do not assume knowledge or imply there are things that somebody should know. We strive to approach all tasks with the enthusiasm of a newcomer because we believe that newcomers are just as valuable in this effort as experts.
 - We consistently make the effort to actively recognize and validate multiple types of contributions.
-- We are always willing to offer help or guidance.
+- We work to offer help and guidance when we are able to do so.
 
 **In times of conflict:**
 
 - We listen.
 - We clearly communicate while acknowledging others’ feelings.
 - We admit when we're wrong, apologize, and accept responsibility for our actions.
-- We are continuously seeking to improve ourselves and our community.
-- We keep our community respectful and open.
+- We are continuously seeking to improve ourselves and the community.
+- We keep the community respectful and open.
 - We make everyone feel heard.
 - We are mindful and kind in our interactions.
 
-**In the future:**
-
-- The future is now.
-
 ## 🌍 Overview
 
-### Our Pledge Pt.1: Fundamentals
-
-We, as members, contributors, and leaders, pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+This Code of Conduct is designed to apply to behavior within the ml5 community and to the uses of the ml5.js library. Members of the ml5 community are expected to follow the Code of Conduct when participating in the [community spaces described below](). Software that uses the ml5.js library is required to follow the Code of Conduct by the library’s copyright license.
```

Comment:
Copy update: empty link
* `[community spaces described below]()` -- missing link? // not sure if there's a list below?

Reply:
Oh we can ref to L45 https://github.com/ml5js/Code-of-Conduct/pull/25/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R45

Reply:
Thanks Joey! I think I prefer to link to the "Scope" section rather than a specific line number. Added this in https://github.com/ml5js/Code-of-Conduct/pull/25/commits/7c03c6c8ad1d0ffde7146a8e46557d21f9495fbf.